src/fedora/30/amd64/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN dnf install -y \
         python2-lldb \
         sudo \
         which \
+        glibc-langpack-en \
```

Comment:
Please alphabetize, it is a [Dockerfile best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments)